packages/checkbox/src/vaadin-checkbox.js

```diff
@@ -82,6 +82,7 @@ class Checkbox extends LabelMixin(
           display: grid;
           grid-template-columns: auto 1fr;
           align-items: baseline;
+          overflow: hidden;
```

Comment:
Without this line, there is a vertical scrollbar on Windows when group does not have a label:

![Screenshot 2022-03-02 at 14 28 57](https://user-images.githubusercontent.com/10589913/156361768-66704275-a476-46ba-807c-5c0286d157ee.png)
e